cmd/aro/mirror.go

```diff
@@ -155,6 +155,17 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 		}
 	}
 
+	for _, ref := range []string{
+		"quay.io/hawkieowl/managed-upgrade-operator:v0.1.795-1b086e9",
```

Comment:
Doesn't MUO provide an image already? I would like to avoid having more dependencies on personal accounts than we already have.